docs/index.md

```diff
@@ -77,7 +56,20 @@ $ redis-server --loadmodule /path/to/redisbloom.so INITIAL_SIZE 400 ERROR_RATE 0
 The default error rate is `0.01` and the default initial capacity is `100`.
 
 ## Bloom vs. Cuckoo
-
 Bloom Filters typically exhibit better performance and scalability when inserting
 items (so if you're often adding items to your dataset then Bloom may be ideal),
 whereas Cuckoo Filters are quicker on check operations and also allow deletions.
+
+## Clients
```

Comment:
can you use the same structure as we use for redisgraph.io?  A table with license and author?

Reply:
:+1:

Reply:
Love it